src/listener.ts

```diff
@@ -0,0 +1,24 @@
+import { NestFactory } from '@nestjs/core';
+import { MicroserviceOptions, Transport } from '@nestjs/microservices';
+import { AppModule } from './app.module';
+
+/**
+ * It creates a microservice that listens to a RabbitMQ queue and uses the AppModule to handle incoming messages
+ */
+async function bootstrap() {
+  const app = await NestFactory.createMicroservice<MicroserviceOptions>(
+    AppModule,
+    {
+      transport: Transport.RMQ,
+      options: {
+        urls: [process.env.RABBITMQ_URL],
+        queue: process.env.RABBITMQ_QUEUE,
+        queueOptions: {
+          durable: false,
```

Comment:
You need to change this to true, because the core service it is listening to is set to true, else it is throwing an error

Reply:
i set it to true before.. it threw error saying i should set it to false.. let me find a away around it

Reply:
![Screenshot from 2022-08-29 14-58-47](https://user-images.githubusercontent.com/88540648/187218681-eecb4b8c-47fc-4e83-904d-45e82dbd0a53.png)

Reply:
Hmmm, it throws error on my side if set to false, true works

Reply:
wahala